src/main/steps/applicant/yourdocuments/alldocuments/alldocuments/tasklistalldocuments.ts

```diff
@@ -38,9 +40,83 @@ export const getOrdersFromCourt = (sectionTitles, taskListItems, userCase, url)
 };
 
 export const getApplicantDocuments = (sectionTitles, taskListItems, userCase, isApplicant) => {
-  let url = `${URL.APPLICANT}${URL.YOUR_WITNESS_STATEMENTS}`;
+  let url = URL.APPLICANT;
   if (!isApplicant) {
-    url = `${URL.RESPONDENT}${URL.YOUR_WITNESS_STATEMENTS}`;
+    url = URL.RESPONDENT;
+  }
+
+  let isDrugDocUploaded = false,
+    isPaternityDocUploaded = false,
+    isPreviousOrdersSubmitted = false,
+    isMedicalReportsUploaded = false,
+    isLettersFromSchool = false,
+    isMedicalRecordsUpload = false,
+    isDigitalDownloadsUploaded = false,
+    isPoliceReportUploaded = false,
+    isWitnessAvailabilityUploaded = false,
+    isTenancyUploaded = false;
+
+  for (const doc of userCase?.citizenUploadedDocumentList || []) {
+    if (
+      doc.value.documentType === applicant_tasklist_items_all_docs_en.drug_alcohol_tests &&
+      doc.value.isApplicant === 'Yes'
```

Comment:
Can we use enum 'YesOrNo' here and all other occurrences in this file?

Reply:
replaced with enum usage